### DISCUSSION TITLE
"Try it" section is showing avobe the rest of the page

### DISCUSSION THREAD

**Discussion by sponsCA** [Azure/api-management-developer-portal#2563] [Feature request]
## Bug description

When you click on "Try It", the section is displaying above the rest of the page, making the text unreadable. (instructions, parameters/headers descriptions, etc.)

## Reproduction steps

1. Go to any API call
2. Click on "Try It"
3. See problem

## Expected behavior

The section should not be above the rest of the page but to the right of it, resizing the whole dom.

## Is your portal managed or self-hosted?

Managed

**Comment by mikebudzynski** [comment]
@sw-spons I marked this Issue as a feature request. It was a conscious design choice to have the console open on top of the API reference to save space.

**Comment by sponsCA** [comment]
> @sw-spons I marked this Issue as a feature request. It was a conscious design choice to have the console open on top of the API reference to save space.

Thanks Mike. By any chance, can you tell us if you're planning to implement the feature and if so, do you have an estimated ETA?

Have a great week-end!

**Comment by mikebudzynski** [comment]
At this moment we won't be able to prioritize development of this feature, unless there's a strong need from the community.
